convert/convertBooks.ts

```diff
@@ -394,7 +402,6 @@ export async function convertBooks(
         for (const book of collection.books) {
             let bookConverted = false;
             switch (book.type) {
-                case 'story':
```

Comment:
I know that it will fall to the default case, but I think it would be better to include `case 'story':` above `default:`